strapi/strapi
# fix(i18n): Add some space to show the Locale values in the CM

### What does it do?

It adds more space (width) to show the locale content in the CM list view

### Why is it needed?

To solve an issue we have with the Locale column in the CM list view, the content is truncated with a 2rem width

### How to test it?

 - Create a new strapi project
 - Create a content type with i18n
 - Add an item to the content type
 - Check the content type overview table

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/19790 and https://github.com/strapi/strapi/issues/20021
